Controllers/ValuesController.cs

```diff
@@ -3,99 +3,78 @@
 using System.Security.Claims;
 using System.Threading.Tasks;
 using Database.Models;
-using incrementally.Services;
+using incrementally_backend.Application;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
-using SqlKata.Compilers;
 
 namespace incrementally.Controllers
 {
-  [Route("api")]
-  [ApiController]
-  [Authorize]
-  public class ValuesController : ControllerBase
-  {
-    private readonly CosmosDbService _cosmosDbService;
-    private readonly PostgresCompiler _compiler;
-    public ValuesController(CosmosDbService cosmosDbService, PostgresCompiler compiler)
+    [Route("api")]
+    [ApiController]
+    [Authorize]
+    public class ValuesController : ControllerBase

```

Comment:
Two comments regarding this controller:
- ValuesController seems like a rather generic name for a controller, and it seems like the Controller offers both app-health endpoints, and functionality for the storage and fetching of recording data. To me it seems convenient to split these. I'm not sure about convention in c#, but in Spring-boot health and readiness are automatically available through a very commonly used spring artifact.

- adding versioning to endpoints, ("api/v1") allows for future flexibility with api versioning in a simple manner.

Reply:
I've added the API versioning as a separate task in the backlog. As for the naming of the ValueController, I can suggest naming it RecordingController? 😄